vue/components/ui/molecules/image-input/image-input.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+    <upload-area
+        class="image-input"
+        v-bind:files.sync="files"
+        v-bind:multiple="multiple"
+        v-bind:accept="'image/*'"
+        v-on:update:files="loadFiles"
+    >
+        <template v-slot="{ openModal }">
+            <div class="image-input-content" v-on:click="openModal">
+                <icon
+                    class="upload-icon"
+                    v-bind:icon="'add-camera'"
+                    v-bind:height="24"
+                    v-bind:width="24"
+                    v-show="!hasImages"
+                />
+                <div
+                    class="image-container"
+                    v-for="(image, index) in imagesData"
+                    v-bind:key="image.name"
+                    v-on:click.stop
+                >
+                    <lightbox
+                        class="image-preview"
+                        v-bind:image="image.data"
+                        v-bind:visible="visibleLightbox === index"
+                        v-on:close="onCloseLightbox"
+                        v-on:click.stop="onClickLightbox(index)"

```

Comment:
Why the stop?

Reply:
So it doesn't propagate to `upload-area` click which would call `openModal` and `onClickLightbox` on a single click